components/dropdown/dropdown-content-mixin.js

```diff
@@ -403,6 +417,20 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 		}
 	}
 
+	__disconnectResizeObserver(entries) {
+		for (let i = 0; i < entries.length; i++) {
+			const entry = entries[i];
+			if (this.__resizeObserver && entry.contentRect.height !== 0) {
+				this.__resizeObserver.disconnect();
+				// wrap in rAF for Firefox
+				requestAnimationFrame(() => {
+					this.__position();
```

Comment:
Should we confirm the dropdown is still open right before we call this?  In case the resize is because they're closing the dropdown and then this is unnecessary?